cask-cli.el

```diff
@@ -247,10 +278,17 @@ The output is formatted as a colon path."
 
 That is packages that have a more recent version available for
 installation."
+  (cask-cli--elpa-dependencies-exist-or-error)
   (-when-let (outdated (cask-outdated (cask-cli--bundle)))
     (princ "Outdated packages:\n")
     (-each outdated 'cask-cli--print-upgrade)))
 
+(defun cask-cli/missing ()
+  "Print a list of direct dependencies which are missing."
+  (-when-let (missing (cask-cli--missing-elpa-dependencies))
+    (princ "Missing packages:\n")
```

Comment:
I think we should remove this header. Like @lunaryorn said, it's easier to grep on a "clean" list.